platform/commonUI/general/src/directives/MCTSplitter.js

```diff
@@ -61,6 +61,7 @@ define(
                         if (initialPosition !== newPosition) {
                             mctSplitPane.position(newPosition);
                         }
+
                     },
```

Comment:
There are no changes in this file and so it should be removed from the pull request.